test/unit_inbroker_parallel_SUITE.erl

```diff
@@ -574,6 +575,42 @@ login_of_passwordless_user1(_Config) ->
     passed.
 
 
+set_tags_for_passwordless_user(Config) ->
+    passed = rabbit_ct_broker_helpers:rpc(Config, 0,
+      ?MODULE, set_tags_for_passwordless_user1, [Config]).
+
+set_tags_for_passwordless_user1(_Config) ->
+    Username = <<"set_tags_for_passwordless_user">>,
+    Password = <<"set_tags_for_passwordless_user">>,
```

Comment:
I think this should be

```
Password = <<"">>,
```

otherwise it's not a password-less user...

Reply:
There was supposed to be a line that clears it 🤦‍♂️ .

Reply:
Corrected.